Stripe/STPPaymentContext.m

```diff
@@ -205,6 +201,13 @@ - (void)setDelegate:(id<STPPaymentContextDelegate>)delegate {
     }];
 }
 
+- (void)setPrefilledInformation:(STPUserInformation *)prefilledInformation {
+    _prefilledInformation = prefilledInformation;
+    if (prefilledInformation.shippingAddress) {
+        self.shippingAddress = prefilledInformation.shippingAddress;
```

Comment:
I'm not sure you want to clobber `self.shippingAddress` here (it probably should only be set from a successful view of the VC?). At minimum setting the prefil shouldn't overwrite if we already have a shipping address set.

Suggested flow: A nil `_shippingAddress` until it is confirmed by the user (you can use this to track verification state). Possibly override the getter so that if the instance var is nil, it returns `self.prefilledInformation.shippingAddress`